documentation/architecture.drawio

```diff
@@ -0,0 +1,335 @@
+          <mxGeometry x="960" y="730" width="180" height="30" as="geometry" />
+        </mxCell>
+        <mxCell id="mH02m_fli5AvtXu53zwJ-29" value="External Components" style="rounded=0;whiteSpace=wrap;html=1;fillColor=#DEDEDE;gradientDirection=north;strokeColor=#6c8ebf;" parent="1" vertex="1">
+          <mxGeometry x="960" y="770" width="180" height="30" as="geometry" />
+        </mxCell>
+        <mxCell id="qYgubVpo4Vi_oRc76slj-39" value="EurDaT Client&lt;br&gt;Dataspace Connector" style="whiteSpace=wrap;html=1;aspect=fixed;fontSize=13;fillColor=#dae8fc;strokeColor=#6c8ebf;" parent="1" vertex="1">
+          <mxGeometry x="510" y="530" width="90" height="90" as="geometry" />
+        </mxCell>
+        <mxCell id="qYgubVpo4Vi_oRc76slj-38" value="Skyminder Client" style="whiteSpace=wrap;html=1;aspect=fixed;fontSize=13;fillColor=#dae8fc;strokeColor=#6c8ebf;" parent="1" vertex="1">
+          <mxGeometry x="410" y="530" width="90" height="90" as="geometry" />
+        </mxCell>
+        <mxCell id="qYgubVpo4Vi_oRc76slj-40" value="" style="ellipse;whiteSpace=wrap;html=1;aspect=fixed;fontSize=13;" parent="1" vertex="1">
+          <mxGeometry x="660" y="570" width="10" height="10" as="geometry" />
+        </mxCell>
+        <mxCell id="zA1G0P91OJ-XDcVy-FAF-8" value="" style="endArrow=none;startArrow=classic;html=1;rounded=0;fontSize=9;entryX=0;entryY=0.5;entryDx=0;entryDy=0;startFill=1;" edge="1" parent="1" source="qYgubVpo4Vi_oRc76slj-7" target="qYgubVpo4Vi_oRc76slj-40">
+          <mxGeometry width="50" height="50" relative="1" as="geometry">
+            <mxPoint x="634.0800000000002" y="578.06" as="sourcePoint" />
+            <mxPoint x="690.000000000018" y="575" as="targetPoint" />
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="zA1G0P91OJ-XDcVy-FAF-11" value="" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;strokeColor=default;entryX=0.5;entryY=0;entryDx=0;entryDy=0;entryPerimeter=0;exitX=0.5;exitY=1;exitDx=0;exitDy=0;exitPerimeter=0;" edge="1" parent="1" source="zA1G0P91OJ-XDcVy-FAF-19" target="zA1G0P91OJ-XDcVy-FAF-1">
+          <mxGeometry relative="1" as="geometry">
+            <mxPoint x="380.0000000000002" y="430.0000000000002" as="sourcePoint" />
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="zA1G0P91OJ-XDcVy-FAF-14" value="" style="edgeStyle=orthogonalEdgeStyle;rounded=0;orthogonalLoop=1;jettySize=auto;html=1;strokeColor=default;exitX=0.25;exitY=1;exitDx=0;exitDy=0;exitPerimeter=0;" edge="1" parent="1" source="zA1G0P91OJ-XDcVy-FAF-19" target="qYgubVpo4Vi_oRc76slj-45">
+          <mxGeometry relative="1" as="geometry">
+            <mxPoint x="340" y="470" as="sourcePoint" />
+          </mxGeometry>
+        </mxCell>
+      </root>
+    </mxGraphModel>
+  </diagram>
+</mxfile>
```

Comment:
Im Bild gibt es auf 2 Netzwerkpfeilen weiße Kreise. Sind diese um zu verdeutlichen das hier Systemgrenzen sind oder was stellen die da? Vielleicht sollte man die in die Legende aufnehmen.

Reply:
Habe die Kreise entfernt - ich denke die schaffen keinen Mehrwert.